crates/db-arch/README.md

```diff
@@ -0,0 +1,11 @@
+# db-arch
+
+## Design
+
+db-arch is designed as the collection for cpu architecture (x86 and arm) related crates. We define CPU architecture specified behaviour for VMM to use in this crate.
+
+## Crate List
+This repository contains the following crates:
+| Name | Description |
+| --- | --- |
+| [cpuid](/crates/db-arch/src/x86/cpuid) | cpuid filter for vmm |
```

Comment:
Could we build `cpuid` as a submodule instead of a create? A hierarchy workspace is too heavy.